runner.go

```diff
@@ -123,7 +124,7 @@ no_proxy = %s
 }
 
 var cmd *exec.Cmd
-
+var tail *exec.Cmd
```

Comment:
Does this need to be a global or can it just be declared locally in `TailFile()` on line 143?

Reply:
I think tail can be easily moved to the `TailFile()` call. The original splunk call (`var cmd *exec.Cmd`) may be a bit trickier as it looks like the `StartServer()` function is leveraging the cmd struct for the splunk process to confirm if its still running or not so it would need to be passed around a bit to capture it.
 
 https://github.com/openshift/splunk-forwarder-images/blob/3657e37dbbd0f36f99acd5813ad30ac1b944ece5/runner.go#L173

Reply:
Updated